components/badge.css

```diff
@@ -4,15 +4,17 @@
 .badge{
     position: absolute;
     border-radius: 50%;
-    border: 2px solid var(--white);
 }
 .badge-red{
+    border: 2px solid var(--white);
```

Comment:
You can store this value 2px solid var(--white) in your variables file as a badge-border variable for easy access.

Reply:
thanks for the suggestion, Will do the change